sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Rules/Hotspots/DoNotHardcodeCredentialsBase.cs

```diff
@@ -71,6 +73,7 @@ public string CredentialWords
         protected DoNotHardcodeCredentialsBase(System.Resources.ResourceManager rspecResources, IAnalyzerConfiguration analyzerConfiguration)
         {
             this.rule = DiagnosticDescriptorBuilder.GetDescriptor(DiagnosticId, MessageFormat, rspecResources).WithNotConfigurable();
```

Comment:
is `rule` used anymore? the naming is confusing - I see this class only handles S2068, does it handle 2 different rules?

should you update the VB .NET rule too?

Reply:
ok... please use better names
`rule` and `hardCodedPasswordRule` both refer to "hard coded passsword rule"

please use `ruleForBaseClass` and `ruleForTrackers` if that's what they do

or some good names

Reply:
In the end I've made the trackers configurable and we don't need the second rule. It seems to be a better solution.